### PR TITLE
Add classifier_activation param to densenet

### PR DESCRIPTION
**Pull request by ivallesp**
Adding cliassifier_activation parameter to the importable densenets so that logits can be retrieved from the pretrained models

### CODE REVIEW

**Comment by qlzh727** [comment]
Thanks for sending the PR. Keras repo is currently not the source of truth and is mirrored from tensorflow. Please send the PR to tensorflow/tensorflow.

We will send announcement when keras repo is formally in use.
